### PR TITLE
fix: add apikey to the client

### LINKED ISSUE #87: EDC-management: "Unauthorized" when fetching "Contract Definitions"
# Bug Report

## Describe the Bug

`Contract definitions` can not be fetched.
HTTP Error 401 is thrown (Unauthorized).

### Expected Behavior

Contract Definitions are fetched and list of items is shown.

### Observed Behavior

1. Throws error message
2. Does not fetch and show items

## Steps to Reproduce

1. Go to 'Contract Definitions'
5. See error

## Context Information

- Used version
  - [DataDashboard https://github.com/eclipse-edc/DataDashboard/commit/dc4f57d3492706d39a6f7f663e706a52ca073efb]
  - [EDC v0.2.0]
- OS: [Windows/CentOS7]

## Error Information

ContractDefinitionsError

## Detailed Description

Response:

```
[
    {
        "message": "Request could not be authenticated",
        "type": "AuthenticationFailed",
        "path": null,
        "invalidValue": null
    }
]
```

### PR DESCRIPTION
**Pull request by ndr-brt**
## What this PR changes/adds

Set api token on the connector client

## Why it does that

fix oversight

## Further notes

_List other areas of code that have changed but are not necessarily linked to the main feature. This could be method
signature changes, package declarations, bugs that were encountered and were fixed inline, etc._

## Linked Issue(s)

Closes #87 

_Please be sure to take a look at the [contributing guidelines](https://github.com/eclipse-edc/.github/blob/main/CONTRIBUTING.md#submit-a-pull-request) and our [etiquette for pull requests](https://github.com/eclipse-edc/.github/blob/main/contributing/pr_etiquette.md)._

### CODE REVIEW

**Comment by ndr-brt** [comment]
@edwo-dev could you please review this? So I will merge after an approval

**Review by wolf4ood — APPROVED** [review]

**Pull request merged**
